portfolio/src/main/webapp/script.js

```diff
@@ -145,4 +145,93 @@ function dark() {
     document.getElementById('poem_1').style.backgroundColor = 'black';
     document.getElementById('poem_2').style.backgroundColor = 'black';
     document.getElementById('poem_3').style.backgroundColor = 'black';
-}
\ No newline at end of file
+}
+
+var amplitude = 70;
+var left_pos = 600;
+var top_pos = -600;
+var pi = Math.PI;
+var inc = pi / 30;
+var params = new Map();
+params.set("one", 0).set("two", 0).set("three", 0).set("four", 0).set("five", 0);
+params.set("six", 0).set("seven", 0).set("eight", 0).set("nine", 0).set("ten", 0);
+
+
+function fly(left_pos, top_pos, position, f_ratio, s_ratio, str) {
+    var fact = document.getElementById(str);
+    var button = document.getElementById("fact");
+    var play_button = document.getElementById("bubbles");
+    button.style.visibility = "hidden";
+    fact.style.visibility = "visible";
+    play_button.style.visibility = "visible";
+    position += inc;
+    fact.style.left = f_ratio*amplitude * Math.cos(position)* Math.sin(position) + left_pos + "px";
+    fact.style.top = s_ratio*amplitude * Math.sin(position) - top_pos + "px";
+    setTimeout(function go() {fly(left_pos, top_pos, position, f_ratio, s_ratio, str);}, 70);
+}
+
+function bubble(str) {
```

Comment:
Can you find a better variable name than `str`?

In this case the function argument refers to the id of the bubble element, right?

https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers